x-pack/test/functional/config.js

```diff
@@ -519,11 +519,19 @@ export default async function ({ readConfigFile }) {
             cluster: ['monitor', 'manage_index_templates'],
             indices: [
               {
-                names: ['geo_shapes*'],
+                names: ['*'],
                 privileges: ['all'],
```

Comment:
I see it's not changing in this PR, but I would prefer we only give `privileges: ['read', 'view_index_metadata'],` to indices unless it's a very specific case where it needs more than that.  Othewise we won't catch a change which requires more privs.